charts/tidb-operator/values.yaml

```diff
@@ -44,12 +44,12 @@ controllerManager:
   logLevel: 2
   replicas: 1
   resources:
-    limits:
-      cpu: 250m
-      memory: 150Mi
```

Comment:
is this intended? cc @zjj2wry @aylei

Reply:
yes, now dbaas is set to a large value to avoid oom. In the future, resources may be controlled by vpa. https://github.com/pingcap/dbaas/pull/1757

Reply:
got